lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -316,7 +316,8 @@ def execute
       end
 
       def create_tar(cvv, repositories, json)
-        export_prefix = "export-#{cvv['id']}"
+        cv_name = cvv['name'].gsub(/\s.+/, '')
```

Comment:
is the cvv label available here? the label doesn't have spaces but I don't remember if it's passed in on the cvv hash

Reply:
changing the code to this:

`export_prefix = "export-#{cvv['label']}-#{cvv['id']}"`

Resulted in this:

`export--2.tar`

I tried with both 'label' and 'Label'

Reply:
I think you might be able to access it with `cvv[:content_view][:label]` from looking at the [API response](https://github.com/Katello/katello/blob/master/app/views/katello/api/v2/content_view_versions/base.json.rabl#L28), but I could be wrong.